papers/neurips_23/pymarl/README.md

```diff
@@ -46,7 +46,7 @@ You need to install the packages from this repository.
 The `-e` allows you to modify imp_marl packages and use them without reinstalling it.
 
 ```
-pip install -e ../../../ .
+pip install -e ../../../ 
```

Comment:
It now works fine

Reply:
Did you add and remove the dot again? :p

Reply:
See this commit https://github.com/moratodpg/imp_marl/pull/74/commits/47642ac0e8ac5e76dc88ca8002d29f32ca9d5102

Reply:
Yeah, I checked it, and it works well without the dot. We are already providing a directory as argument.